ui/src/components/flows/MultiPanelEditorView.vue

```diff
@@ -33,6 +33,14 @@
     const store = useStore()
     const flow = computed(() => store.state.flow.flow)
 
+    onMounted(() => {
+        store.commit("flow/setIsCreating", false)
+    })
+
+    watch(flow, () => {
+        store.commit("flow/setIsCreating", false)
+    })
```

Comment:
I feel this would be set more obviously in the component that edits workflows: `FlowRoot.vue`

You can add this in the onCreate hook.